response_operations_ui/templates/sign_in.html

```diff
@@ -106,17 +107,17 @@ <h2>Sign in</h2>
             {% endif %}
 
             {{
-                onsInput({
+                onsPassword({
                     "id": "password",
                     "name": "password",
-                    "type": "password",
                     "label": {
-                        "text": "Password"
+                        "text": 'Password'
                     },
                     "attributes": {
-                        "required": true
+                        "autocomplete": 'off'
                     },
-                    "error": error
+                    "showPasswordText": "Show password",
+                    "error": passwordError
```

Comment:
The error changed but the code didn't change with it?  and the required true is gone?  Did you test this as my gut tells me this is wrong >_<

Reply:
It looks like you might be right, the form still blocks it and the correct error comes up, however that is server side as this would deal with it client side. We also lose the red box, I will update it for Adam

Reply:
Argg, onsPassword doesn't pass attributes, so we can use them